tests/conftest.py

```diff
@@ -76,27 +76,31 @@ def remove_db(filename):
           }
 
 @pytest.fixture
-def dep_manager(request):
+def dep_tmpdir(request):
+    # copied from tempdir plugin
+    name = request._pyfuncitem.name
+    name = py.std.re.sub("[\W]", "_", name)
+    return request.config._tmpdirhandler.mktemp(name, numbered=True)
+
+@pytest.fixture
+def dep_manager(request, dep_tmpdir):
+    # TODO: This doesn't work as expected
+    # Migrate to test_dependency.py fixture
     if hasattr(request, 'param'):
+        raise Exception('Did not expect this code to be used... investigate')
```

Comment:
@schettino72 I _think_ this fixture was attempting to inspect the `request.param` attribute which is in use on the fixture parametrization on `test/test_dependency.py#69` and then setup the requested backend. pytest does not currently work like that. Maybe it used to?

To verify I put that exception in just to see if anything hit it. Nothing ever does.

I duplicated this fixture in `tests/test_dependency.py` and moved the backend setup code there.

I think that in prior versions only the default backend was being tested in `tests/test_dependency.py`.

Reply:
good catch. thanks. fixed on master.